README.md

```diff
@@ -70,6 +74,28 @@ You can obtain the path for the virtualenv by:
 poetry env info --path
 ```
 
+## Running augmentation
+The `augment.sh` uses the following parameters from `config.yaml`:
+- `data.original` 
+- augmentation hyperparameters:
+  - `augmentation_type`: `ged`/`edge_mapper`/`base`
+  - `similarity_threshold`
+  - `augmentation_ratio`
+
+[Example augmentation config](https://github.com/attilanagy234/TreeSwap/tree/main/opennmt/experiments/runs/simple_aug_example/config.yaml)
+
+```bash
+# create directory for new experiment
+cd opennmt/experiments/runs
+mkdir new_experiment
+cd new_experiment
+
+# create config file
+vim config.yaml
+
+../../../bash_scripts/augment.sh 
```

Comment:
Could we make the usage a little simpler? It would be great to make it so that anyone can use the augmentation after installation with a simple sh scrip with params:
```bash
augment.sh --source_data_path src_path \
           --target_data_path tgt_path \
           --src_lang src_lang \
           --tgt_lang tgt_lang \
           --aug_ratio 0.5 \
           --output_path /tmp \
                    ...
```
What do you think?

Reply:
I think copying the example config is easier than setting several parameters (we have ~20 config parameters). 
I can change the script to use the example config in the existing folder.

Reply:
Yeah, that would be nice actually. Let's provide an example config and explain in the docs what each parameter does